storage/s3/s3.go

```diff
@@ -20,6 +20,8 @@ type Options struct {
 	Secret              string
 	Access              string
 	Token               string
+	FileCredentialsPath string
```

Comment:
Nit. Do we maybe just want this to be `FileCredentials`? I would think if they're using a credentials file that it wants a path.

Reply:
Yea, I agree, the path in `FileCredentialsPath` is probably redundant. so I'm good with `FileCredentials`. 

@donny-dont where can I update the [drone docs](http://plugins.drone.io/drone-plugins/drone-s3/) for this plugin? Also so that I can we can give an example for `FileCredentials` so that it's super clear to the user.

Reply:
The plugin docs for this one is at https://github.com/drone/drone-plugin-index/blob/master/content/drone-plugins/drone-s3-cache/index.md